dotnet/src/Connectors/Connectors.Memory.AzureAISearch/AzureAISearchVectorRecordStoreOptions.cs

```diff
@@ -39,4 +41,10 @@ public sealed class AzureAISearchVectorRecordStoreOptions<TRecord>
     /// See <see cref="VectorStoreRecordKeyAttribute"/>, <see cref="VectorStoreRecordDataAttribute"/> and <see cref="VectorStoreRecordVectorAttribute"/>.
     /// </remarks>
     public VectorStoreRecordDefinition? VectorStoreRecordDefinition { get; init; } = null;
+
+    /// <summary>
+    /// Gets or sets the json serializer options to use when converting between the data model and the azure ai search record.
+    /// Note that when using the default mapper, you will need to provide the same set of <see cref="JsonSerializerOptions"/> both here and when constructing the <see cref="SearchIndexClient"/>.
+    /// </summary>
```

Comment:
nit: This XML documentation is used when generating API Reference docs, so we need to make sure that we capitalize it correctly:
```suggestion
    /// <summary>
    /// Gets or sets the JSON serializer options to use when converting between the data model and the Azure AI Search record.
    /// Note that when using the default mapper, you will need to provide the same set of <see cref="JsonSerializerOptions"/> both here and when constructing the <see cref="SearchIndexClient"/>.
    /// </summary>
```

Reply:
Thanks, fixed.